src/api/rif-notifier-service/interfaces.ts

```diff
@@ -15,13 +16,13 @@ export abstract class NotifierAPIService extends AbstractAPIService {
     this.path = url
   }
 
-  fetch = (): Promise<NotifierResponse> => this._fetch()
+  fetch = (): Promise<NotifierResponse<any>> => this._fetch()
```

Comment:
prefer propagating the generic rather than typing to `any`
```suggestion
  fetch = <T>(): Promise<NotifierResponse<T>> => this._fetch()
```